packages/flutter/lib/src/cupertino/action_sheet.dart

```diff
@@ -977,6 +977,9 @@ class _ActionButtonParentDataWidget extends ParentDataWidget<_CupertinoAlertActi
         targetParent.markNeedsPaint();
     }
   }
+
+  @override
+  Type get debugTypicalAncestorWidget => _CupertinoAlertActionsRenderWidget;
```

Comment:
Here and elsewhere - does it make any difference to wrap these in asserts so that they're null outside of debug mode?

Reply:
Since they are only called from on assert, these methods should already be tree-shaken out in release mode.